ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/policy-maps-pbr.j2

```diff
@@ -0,0 +1,28 @@
+{# PBR policy-map #}
+{% for policy_map in policy_maps.pbr | arista.avd.natural_sort %}
+!
+policy-map type pbr {{ policy_map }}
+{%     set delimiter = namespace(is_set = false) %}
+{%     for class in policy_maps.pbr[policy_map].classes | arista.avd.default([]) %}
+{%         set class_cli = 'class ' ~ class %}
+{%         if policy_maps.pbr[policy_map].classes[class].index is arista.avd.defined %}
+{%             set class_cli = policy_maps.pbr[policy_map].classes[class].index ~ ' ' ~ class_cli %}
+{%         endif %}
+{%         if delimiter.is_set %}
```

Comment:
Normally we don't go to this extend to output delimiters. A simpler implementation would be to just use `if not loop.first`

Reply:
Thanks for the hint.